ballPointPenguin/adopt-a-tree
# Add "Share on Facebook/Twitter" functionality to end of adoption process #46

This PR provides working Facebook/Twitter share button-style links but is not implemented in the most standard way. 
<img width="386" alt="baf_social_media_screenshot" src="https://cloud.githubusercontent.com/assets/2716636/9151798/28a94b10-3dd6-11e5-8113-a6aa7e3312d4.png">
 
## How it should work
The links below explain how to create dynamic links (with images that look like buttons) for each platform via JavaScript when the page loads.  For whatever reason(s), the JavaScript is not executing and the 'buttons' aren't properly built.  I was able to get these to work in plain html, but one or more of the frameworks involved here is stumping me.

https://developers.facebook.com/docs/plugins/share-button
https://dev.twitter.com/web/tweet-button

## How this code works
I have static pngs for the link images instead of using JavaScript.  I have inserted whitespace in between the images using a method I suspect to be the height of hackery.

## Other notes
1. Please manually test
I tested locally, but it would be great if someone else could make sure it works and looks the way we want it to.  I had to make a lot of assumptions during this work.

2. See if you can get the official JavaScript function-based approach to work 

3. See if you can get the alt text to display
